ckanext/datalocale/templates/tag/index.html

```diff
@@ -0,0 +1,14 @@
+{% ckan_extends %}
+
+{% block tags_list %}
+<ul class="unstyled tag-list">
+  {% block tags_list_inner %}
+    {% for tag in c.page.items %}
+      <li>{{ h.link_to(tag.display_name, h.url_for(controller='package', action='search', tags=tag.name), class_='tag') }}</li>
+    {% endfor %}
+  {% endblock %}
+</ul>
+{% endblock %}
+
+
+
```

Comment:
could you explain in the commit message how this improves things? it's not obvious from just reading the code.